pkg/northbound/gnmi/subscribe_test.go

```diff
@@ -237,6 +285,14 @@ func Test_ErrorDoubleSubscription(t *testing.T) {
 	go func() {
 		err = server.Subscribe(serverFake)
 		assert.NilError(t, err, "Unexpected error doing Subscribe")
+		var response *gnmi.SubscribeResponse
+		select {
+		case response = <-responsesChan:
+			log.Error("Should not be receiving response ", response)
+			t.FailNow()
+		case <-time.After(50 * time.Millisecond):
+			log.Info("$$$ Returning from function")
```

Comment:
If this message is still relevant, maybe change the string to something more descriptive.